### ISSUE TITLE
improve `printf`

### ISSUE THREAD

**Issue by xmcp** [jeffswt/pydatagen#7]
try this:

``` python
f=open('1.txt','w')
from contextlib import redirect_stdout
with redirect_stdout(f):
    print('hello world')
```

or

``` python
f=open('1.txt','w')
print('hello world', file=f)
```

**Comment by jeffswt** [comment]
I will try to integrate `printf` with `fprintf` in functionality.

**Comment by xmcp** [comment]
why not simply use `print` instead?

**Comment by jeffswt** [comment]
Because the integrated `print` function does not provide explicit `CRLF` support, i.e. `printf` does not actually print returns after an operation like `print` does.

**Comment by xmcp** [comment]
``` python
print('here you go, without \\n', end='')
```

**Comment by jeffswt** [comment]
I just don't like it, while writing a printf does not manipulate the coding styles of C/C++.

Either way when I was writing Python I wanted to write `cout` or `printf` and when I write C/C++ I stick my fingers to `print`. Respecting the habits of fellow oiers is a standard and a requirement for me.

**Comment by jeffswt** [comment]
`printf()` function improved with the introduction of 0.1.33.

Fixed since 537217ba8f37b3a03be91cc0761d917a0be0fa1d.
